data/json/items/gun/22.json

```diff
@@ -134,7 +131,6 @@
     "recoil": 5,
     "durability": 9,
     "burst": 20,
-    "reload": 500,
```

Comment:
I had presumed similar to TDI vector it was missed in the migration to magazines but if it's intentional I'll revert that